src/common/Error/styled.js

```diff
@@ -0,0 +1,57 @@
+import styled from "styled-components";
+import { NavLink } from "react-router-dom";
+
+export const Wrapper = styled.div`
+    margin: 0 auto;
+    max-width: 630px;
+    text-align: center;
+    padding: 16px;
+`;
+
+export const DangerImage = styled.img`
+    @media(max-width: ${({theme}) => theme.breakpoints.xsmall}px) {
+        width: 100px;
+    }
+`
+
+export const Title = styled.h1`
+    margin: 24px 0;
+    font-size: 36px;
+    line-height: 120%;
+
+    @media(max-width: ${({theme}) => theme.breakpoints.xsmall}px) {
+        font-size: 20px;
+        margin: 16px 0;
+    }
+`;
+
+export const Subtitle = styled.h2`
+    margin: 0 auto;
+    max-width: 450px;
+    font-weight: 500;
+    font-size: 22px;
+    line-height: 130%;
+
+    @media(max-width: ${({theme}) => theme.breakpoints.xsmall}px) {
+        font-size: 14px;
+    }
```

Comment:
Five lines of code are enough here. And again without @media.
`max-width: 427px;
    font-weight: 500;
    margin: auto;
    font-size: 22px;
    line-height: 1.3;`